### PR TITLE
KOGITO-1899: BDD tests: Test travel-agency scenario in native mode

### PR DESCRIPTION
**Pull request by Sgitario**
JIRA Ticket: https://issues.redhat.com/browse/KOGITO-1899
Description: Configure Travel Agency tests with native enabled

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by Sgitario** [comment]
I've run the tests in quarkus native with the workaround and they worked fine.

File: test/features/deploy_travel_agency.feature [hunk 1]
```diff
@@ -52,7 +54,32 @@ Feature: Deploy Travel agency service and verify its functionality
...
+	Examples:
+      | native   | minutes | build-request-cpu | 
+      | disabled | 10      | 1                 | 
+
+	# Disabled as long as https://issues.redhat.com/browse/KOGITO-1893  is not merged or https://issues.redhat.com/browse/KOGITO-1917 is not fixed
+	@disabled
+    @native
+    Examples:
+      | native   | minutes | build-request-cpu | 
+      | enabled  | 30      | 4                 |
```
Comment (Sgitario) [inline test/features/deploy_travel_agency.feature:66]: With CPU 4, the build takes more than 20 min. That's why I set 30 min here.
Without CPU, the build takes more than 30 min).

File: test/features/deploy_travel_agency.feature [hunk 2]
```diff
@@ -52,7 +54,32 @@ Feature: Deploy Travel agency service and verify its functionality
...
+    Given Deploy quarkus example service "kogito-travel-agency/travels" with configuration:
+      | config        | native      | <native>                |
+      | config        | persistence | enabled                 |
+      | config        | events      | enabled                 |
+	  | build-request | cpu         | <build-request-cpu>     |
+    And Deploy quarkus example service "kogito-travel-agency/visas" with configuration:
+      | config        | native      | <native>                |
+      | config        | persistence | enabled                 |
+      | config        | events      | enabled                 |
+	  | build-request | cpu         | <build-request-cpu>     |
```
Comment (radtriste) [inline test/features/deploy_travel_agency.feature:78]: indentation ?

File: test/features/deploy_travel_agency.feature [hunk 3]
```diff
@@ -116,3 +143,14 @@ Feature: Deploy Travel agency service and verify its functionality
 	  """
 
     Then Service "travels" with process name "travels" is available
+
+	Examples:
+      | native   | minutes | build-request-cpu | 
+      | disabled | 10      | 1                 | 
+
+	# Disabled as long as https://issues.redhat.com/browse/KOGITO-1893  is not merged or https://issues.redhat.com/browse/KOGITO-1917 is not fixed
+	@disabled
```
Comment (radtriste) [inline test/features/deploy_travel_agency.feature:152]: indentation ?

**Pull request closed without merging**